server/lib/settings.ts

```diff
@@ -410,7 +410,7 @@ class Settings {
           schedule: '0 30 4 * * *',
         },
         'download-sync': {
-          schedule: '0 * * * * *',
+          schedule: '*/60 * * * * *',
```

Comment:
This can be reverted back to 0 to match the the schedule of the other jobs.

Reply:
Ah, yep let me change that.